zwave_js_server/model/controller.py

```diff
@@ -1,6 +1,18 @@
-from typing import Dict, List
+"""Provide a model for the Z-Wave JS controller."""
+from dataclasses import dataclass, field
+from typing import Dict, List, cast
+
 from ..event import EventBase
-from .node import Node
+from ..protocol import ProtocolType, controller as protocol, get_handler
+from .node import Node, NodeEvent
+
+
+@dataclass
+class ControllerEvent:
```

Comment:
We can't distinguish the events in different classes. They all come in at at the root and are forwarded accordingly. They all follow the same format.

What's the benefit from breaking it up into different classes instead of just having an event class?

Reply:
We can use a single class. This is a leftover from before we settled on mapping the models 1:1 to Z-Wave JS.